test cases/linuxlike/11 runpath rpath ldlibrarypath/meson.build

```diff
@@ -0,0 +1,14 @@
+project('runpath rpath ldlibrarypath', 'c')
+
+libsrc = files('lib.c')
+
+subdir('lib1')
+subdir('lib2')
+
+lib2dir = meson.current_build_dir() + '/lib2'
```

Comment:
Since this is a linuxlike test the backend thing should be ok. Test definitely requires mirror because it builds two libraries with the same filename. Will add a comment.